tt_metal/impl/dispatch/kernels/cq_common.hpp

```diff
@@ -124,7 +124,7 @@ void cq_noc_async_write_init_state(uint32_t src_addr, uint64_t dst_addr, uint32_
     constexpr bool multicast_path_reserve = mcast;
     constexpr bool posted = false;
     constexpr bool linked = false;
-    constexpr uint32_t vc = mcast ? NOC_MULTICAST_WRITE_VC : NOC_UNICAST_WRITE_VC;
+    constexpr uint32_t vc = mcast ? NOC_DISPATCH_MULTICAST_WRITE_VC : NOC_UNICAST_WRITE_VC;
```

Comment:
I would make this a subsequent commit for easy regression testing

Reply:
Done